android/gradle.properties

```diff
@@ -11,8 +11,8 @@
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 #Thu Mar 03 15:15:24 EST 2022
-StripeTerminalReactNative_compileSdkVersion=31
-StripeTerminalReactNative_kotlinVersion=1.6.10
+StripeTerminalReactNative_compileSdkVersion=33
+StripeTerminalReactNative_kotlinVersion=1.8.21
```

Comment:
I think we can now delete some code in `build.gradle` as a result of bumping this:

```groovy
// required for Kotlin versions prior to 1.6.20
// https://issuetracker.google.com/issues/217593040
tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
    kotlinOptions {
        freeCompilerArgs += ["-Xjvm-default=all"]
    }
}
```